packages/forms-web-app/src/dynamic-forms/question.js

```diff
@@ -244,16 +245,10 @@ class Question {
 
 	/**
 	 * @param {JourneyResponse} journeyResponse
-	 * @param {Promise<void>} responseToSave
+	 * @param {object} responseToSave
```

Comment:
Looks like this still returns a promise, maybe `Promise<*>` if `patchLPAQuestionnaire` has an unknown response

Reply:
Done